content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,200 @@
+# Bioactivity data profile
+
+[TOC]
+## Main objective
+This recipe shows how to prepare bioactivity data to meet the ChEMBL submission criteria, focusing on data formats, structures, and vocabularies. 
+
+## Graphical overview of the Recipe FAIRification Objectives
+![](https://i.imgur.com/qNuKXQ4.png)
```

Comment:
This image isn't really a diagram representative of the CHEMBL deposition process.

Reply:
image updated